README.md

```diff
@@ -10,13 +10,16 @@ This is a JavaScript library to easily implement a Google Maps map into your Rea
 - [Installation](#installation)
 - [Usage](#usage)
 - [GoogleMapProvider](#googlemapprovider)
+  - [Properties](#properties)
 - [Hooks](#hooks)
   - [useGoogleMap](#usegooglemap)
   - [useAutocomplete](#useautocomplete)
   - [useDirections](#usedirections)
   - [useGeocoder](#usegeocoder)
   - [usePlacesService](#useplacesservice)
   - [useDistanceMatrix](#usedistancematrix)
+  - [useMaxZoomService](#usemaxzoomservice)
+  - [useElevationService](#useelevationservice)
```

Comment:
Is there any logic behind the order of the hooks in the README? Maybe we can use the same order as in the Google Maps Documentation, or just sort them alphabetically.

Reply:
It's the same order as they are listed in the README below and the latest released hook is at the bottom, which makes sense, I think. Can also order them as in the `Google Maps Documentation` - would also be good. 

Alphabetically does not make many sense here, I think.

Reply:
Perfect